cabal-install/src/Distribution/Client/Config.hs

```diff
@@ -1694,14 +1694,7 @@ postProcessRepo lineno reponameStr repo0 = do
     -- Note: the trailing colon is important
     "file+noindex:" -> do
       let uri = remoteRepoURI repo0
-      return $
-        Left $
-          LocalRepo
-            reponame
-            -- Normalization of Windows paths that use @//./@ does not fully
-            -- normalize the path (see filepath#247), but it is still usable.
-            (normalise (uriPath uri))
-            (uriFragment uri == "#shared-cache")
+      return $ Left $ LocalRepo reponame (normalise (uriPath uri)) (uriFragment uri == "#shared-cache")
```

Comment:
I must say I liked the previous formatting more. This is not critical, of course.

Reply:
Reverted the formatting